open_feature_contrib/providers/flagd/evaluation_context_serializer.py

```diff
@@ -0,0 +1,20 @@
+import typing
+
+from open_feature.evaluation_context.evaluation_context import EvaluationContext
+
+
+class EvaluationContextSerializer:
```

Comment:
Should we fold this code into the provider for simplicity? It does not appear to be providing a lot of value, but I could be wrong here.